metabench.cmake

```diff
@@ -6,127 +6,157 @@ cmake_minimum_required(VERSION 3.0)
 
 find_package(Ruby 2.1)
 if(NOT ${RUBY_FOUND})
-    message(WARNING "Ruby >= 2.1 was not found; the metabench.cmake module can't be used.")
-    return()
+  message(WARNING "Ruby >= 2.1 was not found; the metabench.cmake module can't be used.")
```

Comment:
The indentation here was changed to 2 tabs instead of 4. This is inconsistent with the rest of the file, so I suggest removing it.